_data/signed/ObscureXwing

```diff
@@ -0,0 +1,5 @@
+name : Simon Bright ( I run a technical youtube channel in Russian, I help people solve their problems and learn Linux! Also in my free time I compile kernels for Debian 
+with support for F-Sync and Futex2 patchsets, with support for Ashmem and Binder for Anbox, so that people can easily get the opportunity to play on GNU/Linux 
+and at the same time not lose the ability to work! https://www.youtube.com/channel/UCnQfpqJI9wjcqW-4JS8k-Rg/ )
+name : Simon Bright
```

Comment:
Please, just leave lines 4 and 5.

Reply:
Thanks for the help ! I hope now my signature fits the format?